modules/product/src/test/java/com/opengamma/strata/product/index/ResolvedOvernightFutureOptionTradeTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2016 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.product.index;
+
+import static com.opengamma.strata.collect.TestHelper.assertSerialization;
+import static com.opengamma.strata.collect.TestHelper.coverBeanEquals;
+import static com.opengamma.strata.collect.TestHelper.coverImmutableBean;
+import static com.opengamma.strata.collect.TestHelper.date;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.LocalDate;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+import com.opengamma.strata.product.TradeInfo;
+import com.opengamma.strata.product.TradedPrice;
+
+/**
+ * Test {@link ResolvedOvernightFutureOptionTrade}.
+ */
+public class ResolvedOvernightFutureOptionTradeTest {
+
+  private static final ResolvedIborFutureOption PRODUCT = ResolvedIborFutureOptionTest.sut();
```

Comment:
ResolvedOvernightFutureOptionTrade is not tested in this class?

Reply:
Test class updated to test the new object.